recipes/poco/all/conanfile.py

```diff
@@ -1,85 +1,81 @@
-from collections import OrderedDict
-import os
-
 from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-from conans.tools import Version
+from conans.errors import ConanException, ConanInvalidConfiguration
+from collections import namedtuple, OrderedDict
+import os
 
 
 class PocoConan(ConanFile):
     name = "poco"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://pocoproject.org"
     topics = ("conan", "poco", "building", "networking", "server", "mobile", "embedded")
-    exports_sources = "CMakeLists.txt"
-    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "cmake_find_package"
     settings = "os", "arch", "compiler", "build_type"
     license = "BSL-1.0"
     description = "Modern, powerful open source C++ class libraries for building network- and internet-based " \
                   "applications that run on desktop, server, mobile and embedded systems."
-    options = {"shared": [True, False],
-               "fPIC": [True, False],
-               "enable_xml": [True, False],
-               "enable_json": [True, False],
-               "enable_mongodb": [True, False],
-               "enable_pdf": [True, False],
-               "enable_util": [True, False],
-               "enable_net": [True, False],
-               "enable_netssl": [True, False],
-               "enable_netssl_win": [True, False],
-               "enable_crypto": [True, False],
-               "enable_data": [True, False],
-               "enable_data_sqlite": [True, False],
-               "enable_data_mysql": [True, False],
-               "enable_data_odbc": [True, False],
-               "enable_data_postgresql": [True, False],
-               "enable_encodings": [True, False],
-               "enable_jwt": [True, False],
-               "enable_sevenzip": [True, False],
-               "enable_zip": [True, False],
-               "enable_apacheconnector": [True, False],
-               "enable_cppparser": [True, False],
-               "enable_pocodoc": [True, False],
-               "enable_pagecompiler": [True, False],
-               "enable_pagecompiler_file2page": [True, False],
-               "enable_redis": [True, False],
-               "force_openssl": [True, False],
-               "enable_tests": [True, False],
-               "poco_unbundled": [True, False],
-               "cxx_14": [True, False]
```

Comment:
@madebr could you please explain why cxx_14 is no longer required? 

/cc @prsolucoes

Reply:
It is nowhere used, so I removed it.
Same with `poco_unbundled`, which this pr is actually forcing to be `True`.
I found no way to force openssl in the cmake scripts.

Reply:
okay, now makes sense: https://github.com/pocoproject/conan-poco/blob/57accfa3fc92c90dd793db9757bf89ee5e17c0c2/CMakeLists.txt

Reply:
I see,thanks @madebr !!